pydis_site/apps/content/resources/guides/python-guides/vps_services.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Free Hosting Services for Discord Bots
+description: "This article covers the disasdvantages of utilising a free hosting service to run a discord bot. Two commonly used services are discussed below. "
+toc: 2
+---
+
+
+
+## Recommended VPS services
+
+If you need to run your bot 24/7 (with no downtime), you should consider using a virtual private server (VPS). This is a list of VPS services that are sufficient for running Discord bots.
+
+* Europe
+    * [netcup](https://www.netcup.eu/)
+        * Germany & Austria data centres.
+        * Great affiliate program.
+    * [Yandex Cloud](https://cloud.yandex.ru/)
+        * Vladimir, Ryazan, and Moscow region data centres.
+    * [Scaleway](https://www.scaleway.com/)
+        * France data centre.
+    * [Time 4 VPS](https://www.time4vps.eu/)
+        * Lithuania data centre.
+* US
+    * [GalaxyGate](https://galaxygate.net/)
+        * New York data centre.
+        * Great affiliate program.
+* Global
+    * [Linode](https://www.linode.com/)
+    * [Digital Ocean](https://www.digitalocean.com/)
+    * [OVHcloud](https://www.ovhcloud.com/)
+    * [Vultr](https://www.vultr.com/)
+
+
+## Why not to use free hosting services for bots
+While these may seem like nice and free services, it has a lot more caveats than you may think. For example, below discusses the drawbacks of using common free hosting services to host a discord bot.
```

Comment:
How can "below" (preposition) discuss something? I am confused.

Reply:
Ah right, this is still not resolved. Can you change it to "For example, the paragraph below discusses the..."?

Reply:
Updated to " the drawbacks of using common free hosting services to host a discord bot are discussed below."